control/tests/sisotool_test.py

```diff
@@ -27,7 +27,7 @@ def test_sisotool(self):
         assert_array_almost_equal(ax_rlocus.lines[2].get_data(),initial_point_2)
 
         # Check the step response before moving the point
-        step_response_original = np.array([ 0.,          0.02233651,  0.13118374,  0.33078542,  0.5907113,   0.87041549, 1.13038536,  1.33851053,  1.47374666,  1.52757114])
+        step_response_original = np.array([0.    , 0.0217, 0.1281, 0.3237, 0.5797, 0.8566, 1.116 , 1.3261, 1.4659, 1.526 ])
```

Comment:
Why do these values need to be changed?  The system is the same and so shouldn't the step response be the same?

Reply:
The function in scipy that used to be used to generate the time vector, `scipy.signal.ltisys._default_response_times(A, n)` uses `numpy.linspace` without the keyword arg `endpoint=False` so it returns n+1 data points instead of n. The new function does, so it generates only n data points, which is consistent with how other functions in python-control seem to do it. (thus the slightly changed values). Let me know if you have a preference to go back to n+1 datapoints. My thinking is, if you ask for n datapoints, that ought to be how many you get.